model/minsky.cc

```diff
@@ -265,6 +265,7 @@ namespace minsky
     canvas.setItemFocus(model->addGroup(g));
```

Comment:
OK for the moveTo to work on line 268, yes g needs to be added to model. But you can ditch the call to setItemFocus.

Reply:
Also solves the segfault problem in adjustWires when the two groups don't share an ancester (that should probably be fixed too, but not so urgent).